test/test_enclave_wrangler.py

```diff
@@ -0,0 +1,83 @@
+"""Tests
+
+TODO's
+ - 1. Test framework: Current implementation is ad-hoc for purposes of development.
+ - 2. Change from validate to apply, or do both
+"""
+import os
+
+from enclave_wrangler.new_enclave_api import JSON_TYPE, upload_concept, upload_concept_set, upload_draft_concept_set
+
+# todo: replace what I've done here w/ `upload_dataset` eventually if I can. Right now limited by not being able to
+#  delete a container.
+# from enclave_wrangler.dataset_upload import upload_dataset
+
+
+TEST_INPUT_DIR = os.path.join(os.path.dirname(__file__), 'input', 'test_enclave_wrangler')
+
+
+# todo: Can add this test by composing other non-validation tests (then, remove 'PyUnusedLocal' line)
+# todo: can just not create container for now, and test simply (i) add/delete versions, (ii) add/delete concepts
+# noinspection PyUnusedLocal
+def test_dataset_upload(inpath=os.path.join(TEST_INPUT_DIR, 'test_dataset_upload')):
```

Comment:
Due to time, I'm probably not going to work on this wrapper test `test_dataset_upload` until the end, if even.